packages/contracts-bedrock/deploy/015-OptimismMintableERC20FactoryImpl.ts

```diff
@@ -1,15 +1,23 @@
 import { DeployFunction } from 'hardhat-deploy/dist/types'
 import { constants } from 'ethers'
 
-import { assertContractVariable, deploy } from '../scripts/deploy-utils'
+import {
+  assertContractVariable,
+  deploy,
+  getContractFromArtifact,
+} from '../scripts/deploy-utils'
 
 const deployFn: DeployFunction = async (hre) => {
+  const L1StandardBridgeProxy = await getContractFromArtifact(
+    hre,
+    'Proxy__OVM_L1StandardBridge'
+  )
   await deploy({
     hre,
     name: 'OptimismMintableERC20Factory',
-    args: [],
+    args: [L1StandardBridgeProxy.address],
```

Comment:
this should be the L2StandardBridge?

Reply:
No, this is deployed on L1.
https://github.com/bobanetwork/v3-anchorage/blob/8269ff508f6d92481d6e67af0397b352a1cd4ba2/packages/contracts-bedrock/scripts/Deploy.s.sol#L557

Reply:
ok thanks I see, the one for L2 is a predeploy.